scss/elements/checkboxes.scss

```diff
@@ -81,4 +82,24 @@
   &:checked:focus + span::before {
     @include pixelize($checkbox-checked-focus, $colors);
   }
+  &.is-dark {
+    + span {
+      color: $color-white;
+    }
+    // prettier-ignore
+    + span::before { /* stylelint-disable-line no-descending-specificity */
+      color: $color-white;
+    }
+
+    &:checked + span::before {
+      @include pixelize($checkbox-checked-focus, $colors-checkbox-dark);
```

Comment:
Thanks 🙇 

I fixed `pixelize` bugs in #264.
When #264 is merged, could you fix it as follows:
```scss
&.checked + span::before {
  @include pixelize(2px, $checkbox-checked-focus, $colors-checkbox-dark);
}
&:checked:focus + span::before {
  @include pixelize(2px, $checkbox-checked-focus, $colors-checkbox-dark);
  color: $color-white;
}

Reply:
Sure no problem @BcRikko 👍

Reply:
@Ilyeo Thank you for waiting. 🙇 
I merged #264 
Could you fix it? 🙏

Reply:
@BcRikko For sure, It's done. 👍